src/pip/_internal/commands/wheel.py

```diff
@@ -121,6 +121,11 @@ def run(self, options: Values, args: List[str]) -> int:
 
         reqs = self.get_requirements(args, options, finder, session)
 
+        # Since we want to build regular wheels, disable the building of
+        # editable wheels.
+        for req in reqs:
+            req.no_build_editable = True
```

Comment:
I'm super happy with this. The alternative is to pass a similar flag to the preparer and the wheel builder... Not sure which one is better.

Reply:
So I tried the latter approach, introducing a `BuildPurpose` concept. The diff is a little bit larger but it fits better as this is really *not* a concept that belongs to InstallRequirement.